r/react360.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { AppRegistry, Text, View, StyleSheet } from "react-360";
+
+export default class hello_world extends React.Component {
+    render() {
+        return (
+            <View style={styles.panel}>
```

Comment:
Are these elements required or can they be stripped down?

Reply:
AppRegistry, Text and View are required.
AppRegistry (for mapping names and components), View/Text for the actual UI component to display "Hello World".  StyleSheet can be stripped though the UI will be ugly when actually viewed on the VR device

Reply:
So i just stripped StyleSheet and added the styles direct on the View and Text components. So i will need to send a separate pull request for the changes made since this is already merge right?

Reply:
That won't be necessary since the new styles make it more complicated.

Reply:
Well noted! let me revert my branch changes then to reflect this repos master.